app/javascript/components/mentoring/mentor_inbox/conversation_sorter.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export function ConversationSorter({ sortOptions, setSort, sort }) {
+  function handleChange(e) {
+    setSort(e.target.value)
+  }
+
+  return (
+    <div className="conversation-sorter">
+      <label htmlFor="conversation-sorter-sort">Sort</label>
+      <select
+        id="conversation-sorter-sort"
```

Comment:
Can we make these classes instead of ids in case we have multiple things on the same page at some stage pls? There's a few places where they are.